src/poprox_storage/repositories/accounts.py

```diff
@@ -53,6 +53,16 @@ def fetch_account_by_email(self, email: str) -> Optional[Account]:
             return accounts[0]
         return None
 
+    def create_new_account(self, email: str) -> Account:
+        account_tbl = self.tables["accounts"]
+        query = (
+            sqlalchemy.insert(account_tbl)
+            .values(email=email)
+            .returning(account_tbl.c.account_id, account_tbl.c.email)
+        )
+        row = self.conn.execute(query).one_or_none()
+        return Account(account_id=row.account_id, email=row.email)
```

Comment:
work for a new PR, but I can confirm `row.variable` works. The documentation also says that `row["variable"]` works. This should save us from having to use `row[index]` which is more fragile.